DEVX_ROADMAP.md

```diff
@@ -0,0 +1,46 @@
+# Developer Experience Roadmap
+
+This is a list of the upcoming DevX changes in the next 30 days
+
+## JSON RPC
+*[Gateway deprecation] Sui_syncAccountState will be deprecated and removed.
+* [Gateway deprecation] Gateway’s sui_executeTransaction will be replaced by fullnode’s sui_executeTransaction which has an extra parameter that allows users to choose if a method should block or return immediately.
+* Pagination support for the following APIs.
+  * get_objects_owned_by_address
+  * get_objects_owned_by_object
+  * get_recent_transactions
+  * get_transactions_by_input_object
+  * get_transactions_by_mutated_object
+  * get_transactions_by_move_function
+  * get_transactions_from_addr
+  * get_transactions_to_addr
+  * get_events_by_transaction
+  * get_events_by_transaction_module
+  * get_events_by_move_event_struct_name
+  * get_events_by_sender
+  * get_events_by_recipient
+  * get_events_by_object
+  * get_events_by_timerange
+* sui_eventSubscription API that enables subscribing to all transaction events, including ones that don’t generate network activity (github)
+* A sui_call API that emulates transaction execution, but does not commit the same to the network, akin to eth_call in Ethereum.
+
+## SDK (Typescript, Rust)
+* Reliability improvements in Typescript SDK support for sui_publish API
+* Support for sui_subscribeEvent in Typescript SDK
+* Rust SDK will be kept at parity with the Typescript SDK when feasible
+
+# Sui Move
+* Support for dynamic access of child objects to nullify the need to explicitly pass child objects in a transaction as well as to enable implementation of design patterns/collections that require pointer-like parent->child representation.
+* Support for passing strings and object ID's encodings to entry functions
+* [done] Support for passing a vector of objects to entry functions which will open up a variety of flexible use cases wrt payments and coin management.
```

Comment:
[done] means completed? Complete is a better word here, Done refers to food...I cooked it until the food was was done. I worked until the project was finished/completed.

Reply:
if PRs are merged but not yet tagged for release we should not mark them as done.